src/common.c

```diff
@@ -185,6 +185,9 @@ int ofi_ep_bind_valid(struct fi_provider *prov, struct fid *bfid, uint64_t flags
 			return -FI_EBADFLAGS;
 		}
 		break;
+	case FI_CLASS_SRX_CTX:
+		/* Flags can be ignored */
+		break;
```

Comment:
The app is not supposed to use flags for shared context. The current code ensures flags is 0. So this patch has to be dropped from the PR.